src/ClientData/include/ClientData/ModuleData.h

```diff
@@ -62,6 +62,8 @@ class ModuleData final {
   orbit_grpc_protos::ModuleInfo module_info_;
   bool is_loaded_;
   std::map<uint64_t, std::unique_ptr<orbit_client_protos::FunctionInfo>> functions_;
+  absl::flat_hash_map<std::string, orbit_client_protos::FunctionInfo*> name_to_function_info_map_;
```

Comment:
because hashes collide. and with comparing number of possible values of hashes and strings they mathematically have to collide. I do not think there is a correct way of translating a string to uint64_t without a collision.

Reply:
`std::string_view` might work here as well, right?

Reply:
Interesting. I haven't thought about this. It will solve the problem. I will try that.